public/partials/navbar.html

```
@@ -294,20 +328,42 @@
+                            </a>
+                            <a href="/notifications.html" class="flex items-center py-2 px-3 text-gray-700 dark:text-gray-300 hover:bg-gray-100 dark:hover:bg-gray-700 rounded-md text-sm">
+                                <i class="fas fa-bell mr-2 text-teal-500 w-4 text-center"></i><span>Notifications</span>
                             </a>
-                            <a href="/notifications.html" class="flex items-center py-2 px-3 text-gray-700 dark:text-gray-300 hover:bg-gray-100 dark:hover:bg-gray-700 rounded-md">
-                                <i class="fas fa-bell mr-2 text-teal-500"></i><span>Notifications</span>
+                            <a href="/notification-preferences.html" class="flex items-center py-2 px-3 text-gray-700 dark:text-gray-300 hover:bg-gray-100 dark:hover:bg-gray-700 rounded-md text-sm">
+                                <i class="fas fa-sliders-h mr-2 text-teal-500 w-4 text-center"></i><span>Preferences</span>
                             </a>
-                            <button onclick="logout()" class="w-full flex items-center py-2 px-3 text-red-600 dark:text-red-400 hover:bg-red-50 dark:hover:bg-red-900/20 rounded-md mt-2">
-                                <i class="fas fa-sign-out-alt mr-2"></i><span>Logout</span>
+                            <button onclick="logout()" class="w-full flex items-center py-2 px-3 text-red-600 dark:text-red-400 hover:bg-red-50 dark:hover:bg-red-900/20 rounded-md mt-2 text-sm">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)